gradle/plugins/common/src/main/kotlin/junitbuild.java-nullability-conventions.gradle.kts

```diff
@@ -19,9 +19,23 @@ nullaway {
 
 tasks.withType<JavaCompile>().configureEach {
 	options.errorprone {
-		disableAllChecks = true
+		val onJ9 = java.toolchain.implementation.orNull == JvmImplementation.J9
+		if (name == "compileJava" && !onJ9) {
+			disable(
+				"BadImport",
+				"UnnecessaryLambda",
+				"AnnotateFormatMethod",
+				"StringSplitter",
+				"DoNotCallSuggester",
+				"InlineMeSuggester",
+				"ImmutableEnumChecker",
+				"MissingSummary"
```

Comment:
Some of those checks do sound interesting, though, especially `UnnecessaryLambda`.

Reply:
It might be worth adding a TODO comment for such "nice to fix" so somebody could pick them later.